internal/resources/examples_test.go

```diff
@@ -45,7 +45,27 @@ func TestAccExamples(t *testing.T) {
 		{
 			category: "Grafana Enterprise",
 			testCheck: func(t *testing.T, filename string) {
-				testutils.CheckEnterpriseTestsEnabled(t, ">=11.0.0") // Only run on latest version
+				if strings.Contains(filename, "grafana_data_source_config_lbac_rules") {
+					// TODO: Fix the example to work with import.
+					//
+					// It looks like the test setup fails because the resource is imported but the rules don't actually exist, so the following refresh-after-apply that calls the `.Read` method fails.
+					//
+					// examples_test.go:148: Step 1/1 error: Error running post-apply refresh: exit status 1
+					//
+					//   Error: Failed to get LBAC rules
+					//
+					//     with grafana_data_source_config_lbac_rules.test_rule,
+					//     on terraform_plugin_test.tf line 18, in resource "grafana_data_source_config_lbac_rules" "test_rule":
+					//     18: resource "grafana_data_source_config_lbac_rules" "test_rule" {
+					//
+					//   Could not read LBAC rules for datasource "feg5jgkkikg00e": [GET
+					//   /datasources/uid/{uid}/lbac/teams][500]
+					//   getTeamLBACRulesApiInternalServerError {"message":"Validation error, invalid
+					//   format of team LBAC rules"}
+					t.Skip()
+				} else {
+					testutils.CheckEnterpriseTestsEnabled(t, ">=11.0.0") // Only run on latest version
+				}
```

Comment:
@eleijonmarck FYI, the tests have been failing since https://github.com/grafana/terraform-provider-grafana/pull/2077 – looks like either the test setup for examples is not representative or the resource for LBAC rules doesn't properly work with `terraform import`.

Reply:
@spinillos is on this as well: https://github.com/grafana/terraform-provider-grafana/pull/2090

Reply:
I didn't see that 😅

Reply:
@spinillos if you end up merging your PR before mine I'm happy to revert this change and instead rely on your method of skipping tests!

Reply:
@radiohead thx for the ping. i am trying to clean my notifications on this.